components/src/core/components/Chart/types.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * This file is part of OrangeHRM Inc
+ *
+ * Copyright (C) 2020 onwards OrangeHRM Inc
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see  http://www.gnu.org/licenses
+ */
+
+export type Legend = {
+  enable: boolean;
+  align?: 'start' | 'center' | 'end';
+  fullSize?: boolean;
+  position?: 'center' | 'left' | 'top' | 'right' | 'bottom' | 'chartArea';
+};
+
+export type Tooltip = {
+  enable: boolean;
+  position?: 'average' | 'nearest';
+  titleColor?: string | null;
+  backgroundColor?: string | null;
+};
+
+export type Animation = {
+  enable: boolean;
+  loop?: boolean;
+  delay?: number;
+  duration?: number;
+  easing?:
+    | 'linear'
+    | 'easeInSine'
+    | 'easeOutSine'
+    | 'easeInBounce'
+    | 'easeOutBounce';
+};
+
+export type DataPoint = {
+  label: string;
+  value: number;
+  color: string;
+  legend?: {
+    enable?: boolean;
+    order?: number;
+    label?: string | null;
+  };
+  Tooltip?: {
+    enable?: boolean;
+    order?: number;
+    label?: string | null;
+  };
+};
+
+export const COLOR_TART_ORANGE = '#FB5255';
```

Comment:
*

Reply:
Colors defined in spec https://oxd-prod-infinity.orangehrm.com/main-catalog/charts.html